packages/svelte/src/sdk.ts

```diff
@@ -17,4 +18,45 @@ export function init(options: BrowserOptions): void {
   };
 
   browserInit(options);
+
+  detectAndReportSvelteKit();
+}
+
+/**
+ * Adds a global event processor to detect if the SDK is initialized in a SvelteKit frontend,
+ * in which case we add SvelteKit an event.modules entry to outgoing events.
+ * SvelteKit detection is performed only once, when the event processor is called for the
+ * first time. We cannot perform this check upfront (directly when init is called) because
+ * at this time, the HTML element might not yet be accessible.
+ */
+export function detectAndReportSvelteKit(): void {
+  let detectedSvelteKit: boolean | undefined = undefined;
+
+  const svelteKitProcessor: EventProcessor = event => {
+    if (detectedSvelteKit === undefined) {
+      detectedSvelteKit = isSvelteKitApp();
+    }
+    if (detectedSvelteKit) {
+      event.modules = {
```

Comment:
Where does `event.modules` show up? Is it something that is indexed and we can query for?

Reply:
Yup - it displays in event details page, and is indexed - we can query for it in looker.

Reply:
@lforst, it's vaguely documented [here](https://develop.sentry.dev/sdk/event-payloads/)